packages/gestalt/src/Sheet.js

```diff
@@ -117,14 +121,18 @@ function Sheet(props: SheetProps): Node {
     closeOnOutsideClick = true,
     footer,
     heading,
+    onAnimationEnd,
     onDismiss,
     size = 'sm',
     subHeading,
   } = props;
 
   const [showTopShadow, setShowTopShadow] = useState<boolean>(false);
   const [showBottomShadow, setShowBottomShadow] = useState<boolean>(false);
-  const { animationState, onAnimationEnd } = useAnimation();
+  const {
+    animationState: animationStateHook,
```

Comment:
nit: this name is a bit odd, since it makes it sound like this variable is a hook itself. Perhaps stick with the convention you used for the next one, `animationStateFromHook`?

Reply:
OOH I didn't notice that subtlety, yeah 
DONE